packages/api-react/src/services/wallet.ts

```diff
@@ -260,8 +261,8 @@ export const walletApi = apiWithTag.injectEndpoints({
           },
         } = response;
 
-        const pendingBalance = BigInt(unconfirmedWalletBalance) - BigInt(confirmedWalletBalance);
-        const pendingTotalBalance = BigInt(confirmedWalletBalance) + BigInt(pendingBalance);
+        const pendingBalance = new BigNumber(unconfirmedWalletBalance) - new BigNumber(confirmedWalletBalance);
+        const pendingTotalBalance = new BigNumber(confirmedWalletBalance) + new BigNumber(pendingBalance);
```

Comment:
`new BigNumber` is just an object and `+` and `-` operator cannot be overridden.
```typescript
const pendingBalance = new BigNumber(unconfirmedWalletBalance) - new BigNumber(confirmedWalletBalance);
const pendingTotalBalance = new BigNumber(confirmedWalletBalance) + new BigNumber(pendingBalance);
```
should be
```typescript
const pendingBalance = BigNumber(unconfirmedWalletBalance).minus(BigNumber(confirmedWalletBalance));
const pendingTotalBalance = BigNumber(confirmedWalletBalance).plus(BigNumber(pendingBalance));
```

Reply:
Nice catch, fixed